host/app/api.ts

```diff
@@ -0,0 +1,6 @@
+export * from './lib/card-api';
+export { default as string } from './lib/string';
+export { default as textArea } from './lib/text-area';
+export { default as date } from './lib/date';
+export { default as datetime } from './lib/datetime';
+export { default as integer } from './lib/integer';
```

Comment:
We hook into the AMD module that ember-cli creates for us, and reexport the runtime API from this module (which is then leveraged as part of the worker's externals build task)